res/openvpn-gui-res-it.rc

```diff
@@ -17,7 +18,7 @@
  *  along with this program (see the file COPYING included with this
  *  distribution); if not, write to the Free Software Foundation, Inc.,
  *  59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
- */
+*/
```

Comment:
Original looks right -- space removed by mistake?

Reply:
That's probably it. I'll just restore it.